t/units/basic_parser.t

```diff
@@ -150,6 +150,7 @@ subtest 'Check display methods' => sub {
 };
 
 subtest 'Check possible answer format branches' => sub {
+
```

Comment:
Why add the empty line here?  Please remove.

Reply:
The empty line appeared after a perltidy so I left it.  I'm not sure which .perltidyrc was in play when I ran it, but I don't like it either.  I will remove it.

Reply:
I see.  Yeah, different .perltidy files can do quite different things.